code/game/objects/structures/displaycase.dm

```diff
@@ -106,13 +106,25 @@
 	return
 
 /obj/structure/displaycase/attackby(obj/item/weapon/W, mob/user, params)
-	if(W.GetID() && electronics && !broken)
+	if(W.GetID() && !broken)

```

Comment:
So, I ran through some tests and the original changes that I made don't seem to affect player made display cases. You can make a display case that is not openable by anyone and vice versa. If you guys want, I can delete the line of code that makes the captains display case openable.